core/src/main/web/outputdisplay/bko-plot/bko-plot.js

```diff
@@ -236,6 +236,45 @@
           });
         };
 
+        function measureText(pText, pFontSize, pStyle) {
```

Comment:
seems like this should come from a std lib like jquery or angular but i can't find anything.